src/schedule/MongoSchedule.ts

```diff
@@ -42,23 +56,27 @@ export class MongoSchedule extends Schedule {
       this.logger,
       pingIntervalMs,
       this.startAllJobs.bind(this),
+      maxPingRetries,
+      retryIntervalMs,
     );
   }
 
   /**
    * Creates a MongoSchedule that is connected to the MongoDB with the provided url.
    *
    * @param momoOptions configuration options for the connection to establish and the Schedule instance to create
+   * @param maxPingRetries attempts of retrying failed schedule pings before throwing an error (default 1)
+   * @param retryIntervalMs time before retrying a failed schedule ping (default 1000 ms)
    */
-  public static async connect({
-    pingIntervalMs = 60_000,
-    scheduleName,
-    ...connectionOptions
-  }: MomoOptions): Promise<MongoSchedule> {
+  public static async connect(
+    { pingIntervalMs = 60_000, scheduleName, ...connectionOptions }: MomoOptions,
+    maxPingRetries = 1,
```

Comment:
Why are the `maxPingRetries` and `retryIntervalMs` not part of the `MomoOptions` object? I'm not sure they need to be handled differently here. While that is a public interface, extending it with a couple of optional fields shouldn't break anything, right?